tools/SourceKit/CMakeLists.txt

```diff
@@ -500,5 +500,10 @@ include_directories(BEFORE
   ${SOURCEKIT_SOURCE_DIR}/include
 )
 
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
```

Comment:
Pardon my CMake ignorance, but is there a way to phrase this such that it specifies non-Darwin instead? I presume that future Unix-y target platforms will also be in need of these headers.